packages/@sanity/base/package.json

```diff
@@ -44,7 +44,7 @@
     "@sanity/structure": "2.2.3",
     "@sanity/transaction-collator": "2.1.4",
     "@sanity/types": "2.2.3",
-    "@sanity/ui": "^0.32.8",
+    "@sanity/ui": "^0.33.0",
```

Comment:
Maybe we should make sure to give some kind of notice to people that when this is released they have to update to get rid of the missing theme context error? 😊